extensions/wikia/GlobalNavigation/templates/GlobalNavigationAccountNavigation_index.php

```diff
@@ -1,24 +1,50 @@
-<ul id="AccountNavigation" class="AccountNavigation table-cell">
-	<li class="account-navigation-first-item">
-		<div class="links-container<?php if ($isAnon): ?> anon <?php endif; ?>">
-			<?= $navItemLinkOpeningTag ?>
+<ul id="AccountNavigation" class="AccountNavigation table-cell <?= $enableNewAuth ? 'newAuth' : '' ?> ">
+	<?php if ($enableNewAuth): ?>
```

Comment:
What do you think about separating new and old markup and put it in separate template files?